src/vcpkg/install.cpp

```diff
@@ -692,7 +692,7 @@ namespace vcpkg
         std::error_code ec;
 
         auto usage_file = installed.usage_file(bpgh.spec);
-        if (fs.exists(usage_file, IgnoreErrors{}))
+        if (fs.is_regular_file(usage_file))
```

Comment:
Hm it is a supported use case to use a new vcpkg-tool with an old port via versioning?

Reply:
Yes, you will just get a warning if you try to install an old port that was affected by this. Just vcpkg CI will treat it as an error.
Furthermore, I think we shouldn't ever try to support ports that make use of bugs in our tool.

Reply:
Yes, that's why post-build checks no longer fail the run

Reply:
My only concern was that the package has no license file after this change (and that the resulting package changes but the binary hash is the same, but I don't care). 

> Furthermore, I think we shouldn't ever try to support ports that make use of bugs in our tool.

Totally agree. I just wanted to ask :)

Reply:
> My only concern was that the package has no license file after this change (and that the resulting package changes but the binary hash is the same, but I don't care).

That is not possible. The copyright file gets installed by cmake. If cmake installed it, it won't be removed. During post build checks we only check that everything is OK but don't clean anything up if something is broken,

Reply:
> The copyright file gets installed by cmake. If cmake installed it, it won't be removed. 

Sorry my mistake. I only read `install.cpp` and `copyright` and thought this code installed somehow the license files (actually I know that they are installed by cmake)